Mathlib/Geometry/Manifold/ContMDiff/Defs.lean

```diff
@@ -187,7 +194,7 @@ variable (I I') in
 /-- A function is `n` times continuously differentiable in a manifold if it is continuous
 and, for any pair of points, it is `n` times continuously differentiable in the charts
 around these points. -/
-def ContMDiff (n : ℕ∞) (f : M → M') :=
+def ContMDiff (n : WithTop ℕ∞) (f : M → M') :=
```

Comment:
This is about line 203, but I can't comment on that line directly:
"Deducing differentiability from higher differentiability" feels about as logical as "Deducing smoothness from higher smoothness" to me. What do you think?

Reply:
To me, differentiability is specifically about `C^1` smoothness, while "smoothness" is more generic. Since this paragraph deals with a generic smoothness exponent, I'd rather keep the current wording.